web/pdf_page_detail_view.js

```diff
@@ -140,10 +141,21 @@ class PDFPageDetailView extends BasePDFPageView {
       return;
     }
 
-    const { viewport, maxCanvasPixels } = this.pageView;
+    const { viewport } = this.pageView;
```

Comment:
Can't we just do the following here, and avoid any changes in the constructor?
```suggestion
    const { viewport, capCanvasAreaFactor } = this.pageView;
```